documentation/JavaFiddle_Requirements.txt

```diff
@@ -58,5 +58,8 @@ Reviewboard functionality
 - add/edit/read comments
 - approve/reject
 
+[Tsanda Artyom] Ideas:
+- add possibility to import project from an existing one on the local machine
+- add wiget with javadocs where you can search class documentation you want 
```

Comment:
Don't imagine. Does it consider only docs in project? Seems to be pointless, for our target projects won't be huge to need documentation.